client/scss/components/_body_content.scss

```diff
@@ -45,3 +45,27 @@
     }
   }
 }
+
+$large-width: (5/7 * 100)+'%';
+
+.body-content__part--full-width {
```

Comment:
I'd like to see how we use this generically for figure as it does feel very body content specific, but I'll leave that for another PR.